programs/marginfi/src/instructions/marginfi_group/add_pool_permissionless.rs

```diff
@@ -99,7 +99,6 @@ pub fn lending_pool_add_bank_permissionless(
     );
     bank.config.oracle_setup = OracleSetup::StakedWithPythPush;
     bank.config.oracle_keys[0] = settings.oracle;
-    bank.config.update_config_flag(true, PYTH_PUSH_MIGRATED);
```

Comment:
I think we can keep this (staked collateral banks always have the flag set in 0.1.4 or later). Or we set it on StakedSettings and they inherit it through this or propagate.